makeingtables/src/Components/Comments.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import DeleteIcon from '@mui/icons-material/Delete';
+import ImportExportIcon from '@mui/icons-material/ImportExport';
+
+export default class Comments extends React.Component {
+    constructor(props) {
+    super(props);
+  
+    // Initializing the state 
+    this.state = {
+        renderType:'comments',
+        items: [],
+        revData : false
+      };
+  }
+
+  
+  render() {
+    const Comments = this.props.items;
```

Comment:
don't create variable in render , as this.props.items will again and again assign into comments variable,
directly use this.props.items in place of comments variable